packages/dom/src/wc-clone.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Custom deep clone function that replaces Percy's current clone behavior.
+ * This enables us to capture shadow DOM in snapshots. It takes advantage of `attachShadow`'s mode option set to open 
+ * https://developer.mozilla.org/en-US/docs/Web/API/Element/attachShadow#parameters
+ */
+const deepClone = host => {
```

Comment:
how do we know this (not-trivial) walker works ?
do we need any tests ? or is this code copy-pasted from somewhere ?

Reply:
It's copied from https://stackoverflow.com/a/55540552 and modified a bit. I don't think this fork is really set up to run tests.